src/simularium/VisGeometry/index.ts

```diff
@@ -714,6 +703,54 @@ class VisGeometry {
         this.controls.enabled = true;
     }
 
+    private setPdbLods(): void {
+        // set lod for pdbs.
+        this.geometryStore.forEachPDB((agentGeo) => {
+            agentGeo.beginUpdate();
+        });
+
+        let visAgent: VisAgent;
+        let agentData: AgentData;
+        let pdbModel: PDBModel;
```

Comment:
it doesn't seem like you ever reference these outside of the loop, so can't you declare them as `const` down below when you are looping through?

Reply:
I'll make the change to make them const inside the loop.